src/s3ql/oauth_client.py

```diff
@@ -31,8 +34,14 @@ def parse_args(args):
     parser.add_debug()
     parser.add_quiet()
     parser.add_version()
+    parser.add_oauth()
 
-    return parser.parse_args(args)
+    options =  parser.parse_args(args)
+    if options.client_id == '':
```

Comment:
Can you give an example of a situation where you'd want to use a different client id?

Reply:
google drive limit max number of request per minut/day depends of the  application, so maybe you could have an google application with a higher cap.(This is my case)